datascience/tables.py

```diff
@@ -202,6 +202,11 @@ def rows(self):
         return self.Rows(self)
 
     @property
+    def row(self):
+        assert self.num_rows == 1, 'Table has more than one row.'
+        return self.rows[0]
```

Comment:
Done!
`ValueError` does not seem the right choice, as its used to denote an error in input args. Switched to a `RuntimeError` instead.